ui/imports/shared/views/chat/LinksMessageView.qml

```diff
@@ -255,9 +255,11 @@ Column {
             width: linkImage.visible ? linkImage.width + 2 : 300
             height: {
                 if (linkImage.visible) {
-                    return linkImage.height + (Style.current.smallPadding * 2) + linkTitle.height + 2 + linkSite.height
+                    return linkImage.height + ((linkData.thumbnailUrl.indexOf(".gif") === -1) ?
```

Comment:
Better use `foo.endsWith(".gif")`; not only it would be faster but it wouldn't fail on URLs like "foobar.gif.png"

Reply:
I guess it should also be case insensitive, right?

Reply:
Yes, that too

Reply:
done! besides the case insensitive, after talking to @alexjba it seems this will change to lower case anyways